spec/pasteo/local_spec.rb

```diff
@@ -1,32 +1,70 @@
 require 'securerandom'
 
-RSpec.describe Paseto::Local do
-  subject { described_class.from_encode64_key(key, footer) }
+RSpec.describe Paseto::V2::Local do
+  subject { described_class }
 
-  let(:key) { '2eOIs+JWWCKvFDg+eHFsIBHfMuN+3bqkceK8moM4S1Y' }
+  let(:payload) { 'test' }
+  let(:encoded_key) { '2eOIs+JWWCKvFDg+eHFsIBHfMuN+3bqkceK8moM4S1Y' }
+  let(:key) { Paseto::V2::Local::Key.decode64(encoded_key) }
   let(:footer) { nil }
-  let(:encrypted_message) { 'v2.local.6EHOXWuFHUBNy9gEB8sSU5NTF83oMagI/j89rE26Wmk' }
+  let(:token) { 'v2.local.6EHOXWuFHUBNy9gEB8sSU5NTF83oMagI/j89rE26Wmk' }
   let(:nonce) { '6EHOXWuFHUBNy9gE' }
 
   before { allow(described_class).to receive(:generate_nonce).and_return(Paseto.decode64(nonce)) }
 
+  describe Paseto::V2::Local::Key do
+    subject { described_class }
+
+    it '.encode64 returns a base64-encoded key' do
+      expect(key.encode64).to eq(encoded_key)
+    end
+
+    it '.decode64(key.encode64) is a no-op' do
+      key = subject.generate
+      round_trip = subject.decode64(key.encode64)
+      expect(round_trip.instance_variable_get(:@key)).to eq(key.instance_variable_get(:@key))
```

Comment:
So cracking open an instance for assertions is a code smell. Should be able to test its behavior without slipping around in the guts.

Maybe it makes sense to turn the `Key` classes in Structs and get the free equality check (assuming RbNaCl plays nice with `==`)?